site/gatsby-site/src/tailwind.css

```diff
@@ -4,20 +4,28 @@
 
 @layer base {
 
-    h1 {
-        font-size: calc(1.375rem + 1.5vw);
-    }
-
     .h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
         @apply font-medium mb-2 mt-0 leading-tight;
     }
 
+    .h1, h1 {
```

Comment:
I'm not sure about customizing heading styles. Since we are adopting Flowbite, I expect them to look like what they do in Flowbite's docs.

Maybe we are missing their typography plugin? https://github.com/themesberg/flowbite-typography

@clari182, what do you think?

Reply:
@cesarvarela yes, the idea is to eventually remove all custom styles and use Flowbites. Maybe in some cases, there are still some concurrency issues with Tailwind/Bootstrap so this was necessary. But for good practice, it should be removed and fixed in another way.

Reply:
That sounds good, @clari182. Please write that in an issue or something; we can tackle this in a future "tech-debt-paying" week.